libminifi/test/rocksdb-tests/RocksDBStreamTests.cpp

```diff
@@ -45,7 +45,9 @@ TEST_CASE_METHOD(RocksDBStreamTest, "Verify simple operation") {
   std::string content = "banana";
   minifi::io::RocksDbStream outStream("one", gsl::make_not_null(db.get()), true);
   outStream.write(content);
-  REQUIRE(outStream.write(content) > 0);
+  const auto second_write_result = outStream.write(content);
+  REQUIRE(second_write_result > 0);
+  REQUIRE(!minifi::io::isError(second_write_result));
```

Comment:
`REQUIRE_FALSE`?

Reply:
IMO `!` is harder to spot than `REQUIRE_FALSE`, the catch2 library mentions something about not being able to decompose "complex" expressions but I don't think that applies here, ultimately it's fine either way by me

Reply:
changed to REQUIRE_FALSE in 5c6ba7b